downstream/modules/platform/ref-controller-api-config-settings.adoc

```diff
@@ -51,7 +51,7 @@ To compose a named URL for this specific resource object by using `NAMED_URL_FOR
 
 * The first part of the URL format is `<name>`, which indicates that you can find the label resource detail in `/api/v2/labels/5/`, and look for `name` field in returned JSON. 
 If you have the `name` field with value `Foo`, then the first part of the unique identifier is `Foo`.
-* The second part of the format is double pluses "++". 
+* The second part of the format is double pluses: {plus}{plus}. 
```

Comment:
Suggest 'double pluses' -> 'double plus sign' because #1: pluses is anyways plural and you're covering it when you mention 'double', and #2: IBM style guide contains a few usages of plus sign with symbol in parenthesis.